{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>Due to default case-insensitive collation in MySQL or MariaDB databases, third-party authentication user IDs are not case-sensitive and could cause different IDs to match.</p>\n<h3 id=\"patches\">Patches</h3>\n<p>This issue has been addressed by <a href=\"https://github.com/python-social-auth/social-app-django/pull/566\">https://github.com/python-social-auth/social-app-django/pull/566</a> and fix released in 5.4.1.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<p>An immediate workaround would be to change collation of the affected field:</p>\n<pre><code class=\"language-mysql\">ALTER TABLE `social_auth_association` MODIFY `uid` varchar(255) COLLATE `utf8_bin`;\n</code></pre>\n<h3 id=\"references\">References</h3>\n<p>This issue was discovered by folks at <a href=\"https://opencraft.com/\">https://opencraft.com/</a>.</p>\n<h3 id=\"references-1\">References</h3>\n<ul>\n<li><a href=\"https://github.com/python-social-auth/social-app-django/security/advisories/GHSA-2gr8-3wc7-xhj3\">https://github.com/python-social-auth/social-app-django/security/advisories/GHSA-2gr8-3wc7-xhj3</a></li>\n<li><a href=\"https://github.com/python-social-auth/social-app-django/pull/566\">https://github.com/python-social-auth/social-app-django/pull/566</a></li>\n<li><a href=\"https://github.com/python-social-auth/social-app-django/commit/31c3e0c7edb187004d8abbde7e9c4f7ef9098138\">https://github.com/python-social-auth/social-app-django/commit/31c3e0c7edb187004d8abbde7e9c4f7ef9098138</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-2gr8-3wc7-xhj3\">https://github.com/advisories/GHSA-2gr8-3wc7-xhj3</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-2gr8-3wc7-xhj3",
            "title": "[social-auth-app-django] social-auth-app-djangon affected by Improper Handling of Case Sensitivity",
            "date_modified": "2024-04-24T18:47:22.000Z",
            "date_published": "2024-04-24T18:47:21.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-2gr8-3wc7-xhj3"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Versions of the package onnx before and including 1.15.0 are vulnerable to Out-of-bounds Read as the ONNX_ASSERT and ONNX_ASSERTM functions have an off by one string copy.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2024-27319\">https://nvd.nist.gov/vuln/detail/CVE-2024-27319</a></li>\n<li><a href=\"https://github.com/onnx/onnx/commit/08a399ba75a805b7813ab8936b91d0e274b08287\">https://github.com/onnx/onnx/commit/08a399ba75a805b7813ab8936b91d0e274b08287</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FGTBH5ZYL2LGYHIJDHN2MAUURIR5E7PY\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FGTBH5ZYL2LGYHIJDHN2MAUURIR5E7PY</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/TFJJID2IZDOLFDMWVYTBDI75ZJQC6JOL\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/TFJJID2IZDOLFDMWVYTBDI75ZJQC6JOL</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-h8wv-9h96-m4hr\">https://github.com/advisories/GHSA-h8wv-9h96-m4hr</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-h8wv-9h96-m4hr",
            "title": "[onnx] Onnx Out-of-bounds Read vulnerability",
            "date_modified": "2024-04-24T14:38:10.000Z",
            "date_published": "2024-02-23T18:30:59.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-h8wv-9h96-m4hr"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Versions of the package onnx before and including 1.15.0 are vulnerable to Directory Traversal as the external_data field of the tensor proto can have a path to the file which is outside the model current directory or user-provided directory. The vulnerability occurs as a bypass for the patch added for CVE-2022-25882.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2024-27318\">https://nvd.nist.gov/vuln/detail/CVE-2024-27318</a></li>\n<li><a href=\"https://github.com/onnx/onnx/commit/66b7fb630903fdcf3e83b6b6d56d82e904264a20\">https://github.com/onnx/onnx/commit/66b7fb630903fdcf3e83b6b6d56d82e904264a20</a></li>\n<li><a href=\"https://security.snyk.io/vuln/SNYK-PYTHON-ONNX-2395479\">https://security.snyk.io/vuln/SNYK-PYTHON-ONNX-2395479</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FGTBH5ZYL2LGYHIJDHN2MAUURIR5E7PY\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FGTBH5ZYL2LGYHIJDHN2MAUURIR5E7PY</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/TFJJID2IZDOLFDMWVYTBDI75ZJQC6JOL\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/TFJJID2IZDOLFDMWVYTBDI75ZJQC6JOL</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-whh8-fjgc-qp73\">https://github.com/advisories/GHSA-whh8-fjgc-qp73</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-whh8-fjgc-qp73",
            "title": "[onnx] Onnx Directory Traversal vulnerability",
            "date_modified": "2024-04-24T14:35:49.000Z",
            "date_published": "2024-02-23T18:30:59.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-whh8-fjgc-qp73"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>A remote Matrix user with malicious intent, sharing a room with Synapse instances before 1.105.1, can dispatch specially crafted events to exploit a weakness in how the auth chain cover index is calculated. This can induce high CPU consumption and accumulate excessive data in the database of such instances, resulting in a denial of service.</p>\n<p>Servers in private federations, or those that do not federate, are not affected.</p>\n<h3 id=\"patches\">Patches</h3>\n<p>Server administrators should upgrade to 1.105.1 or later.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<p>One can:</p>\n<ul>\n<li>ban the malicious users or ACL block servers from the rooms; and/or</li>\n<li>leave the room and purge the room using the admin API</li>\n</ul>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>If you have any questions or comments about this advisory, please email us at <a href=\"mailto:security@element.io\">security AT element.io</a>.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/element-hq/synapse/security/advisories/GHSA-3h7q-rfh9-xm4v\">https://github.com/element-hq/synapse/security/advisories/GHSA-3h7q-rfh9-xm4v</a></li>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2024-31208\">https://nvd.nist.gov/vuln/detail/CVE-2024-31208</a></li>\n<li><a href=\"https://github.com/element-hq/synapse/commit/55b0aa847a61774b6a3acdc4b177a20dc019f01a\">https://github.com/element-hq/synapse/commit/55b0aa847a61774b6a3acdc4b177a20dc019f01a</a></li>\n<li><a href=\"https://github.com/element-hq/synapse/releases/tag/v1.105.1\">https://github.com/element-hq/synapse/releases/tag/v1.105.1</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/matrix-synapse/PYSEC-2024-50.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/matrix-synapse/PYSEC-2024-50.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-3h7q-rfh9-xm4v\">https://github.com/advisories/GHSA-3h7q-rfh9-xm4v</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-3h7q-rfh9-xm4v",
            "title": "[matrix-synapse] Synapse V2 state resolution weakness allows Denial of Service (DoS)",
            "date_modified": "2024-04-24T14:32:56.000Z",
            "date_published": "2024-04-23T21:15:46.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-3h7q-rfh9-xm4v"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h3 id=\"summary\">Summary</h3>\n<p>A vulnerability has been discovered in the handling of the referrer header in the application, which could allow an attacker to conduct open redirects. The issue arises from improper validation of the referrer header in certain conditions. By manipulating the referrer header, an attacker could potentially redirect users to malicious websites, phishing pages, or other dangerous destinations.</p>\n<h3 id=\"poc\">PoC</h3>\n<p>If you change the referer header, you will be redirected to that domain without verifying.</p>\n<p><a href=\"https://github.com/Clinical-Genomics/cg/blob/master/cg/server/invoices/views.py#L173\">https://github.com/Clinical-Genomics/cg/blob/master/cg/server/invoices/views.py#L173</a></p>\n<h3 id=\"impact\">Impact</h3>\n<p>An attacker exploiting this vulnerability could trick users into visiting malicious websites or disclose sensitive information by redirecting them to unintended destinations. This could lead to various attacks including phishing, malware distribution, or further exploitation of other vulnerabilities.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/Clinical-Genomics/cg/security/advisories/GHSA-w228-rfpx-fhm4\">https://github.com/Clinical-Genomics/cg/security/advisories/GHSA-w228-rfpx-fhm4</a></li>\n<li><a href=\"https://github.com/Clinical-Genomics/cg/commit/96e6a968a5a3639cc40ad251ad65952e4f38dd25\">https://github.com/Clinical-Genomics/cg/commit/96e6a968a5a3639cc40ad251ad65952e4f38dd25</a></li>\n<li><a href=\"https://github.com/Clinical-Genomics/cg/blob/master/cg/server/invoices/views.py#L173\">https://github.com/Clinical-Genomics/cg/blob/master/cg/server/invoices/views.py#L173</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-w228-rfpx-fhm4\">https://github.com/advisories/GHSA-w228-rfpx-fhm4</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-w228-rfpx-fhm4",
            "title": "[cg] cg vulnerable to an Open Redirect Vulnerability on Referer Header",
            "date_modified": "2024-04-23T16:21:24.000Z",
            "date_published": "2024-04-23T16:21:22.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-w228-rfpx-fhm4"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Buffer overflow in the C implementation of the <code>apply_delta</code> function <code>in _pack.c</code> in Dulwich before 0.9.9 allows remote attackers to execute arbitrary code via a crafted pack file.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-0838\">https://nvd.nist.gov/vuln/detail/CVE-2015-0838</a></li>\n<li><a href=\"https://lists.launchpad.net/dulwich-users/msg00829.html\">https://lists.launchpad.net/dulwich-users/msg00829.html</a></li>\n<li><a href=\"http://www.debian.org/security/2015/dsa-3206\">http://www.debian.org/security/2015/dsa-3206</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vjjf-3rvg-gv3v\">https://github.com/advisories/GHSA-vjjf-3rvg-gv3v</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vjjf-3rvg-gv3v",
            "title": "[dulwich] Dulwich Buffer Overflow when handling pack files",
            "date_modified": "2024-04-22T22:58:51.000Z",
            "date_published": "2022-05-17T04:14:09.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vjjf-3rvg-gv3v"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.8.x before 1.8.16, 1.9.x before 1.9.11, and 1.10.x before 1.10.3 use a hardcoded password for a temporary database user created when running tests with an Oracle database, which makes it easier for remote attackers to obtain access to the database server by leveraging failure to manually specify a password in the database settings TEST dictionary.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-9013\">https://nvd.nist.gov/vuln/detail/CVE-2016-9013</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/nov/01/security-releases/\">https://www.djangoproject.com/weblog/2016/nov/01/security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3835\">http://www.debian.org/security/2017/dsa-3835</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/94069\">http://www.securityfocus.com/bid/94069</a></li>\n<li><a href=\"http://www.securitytracker.com/id/1037159\">http://www.securitytracker.com/id/1037159</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-3115-1\">http://www.ubuntu.com/usn/USN-3115-1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mv8g-fhh6-6267\">https://github.com/advisories/GHSA-mv8g-fhh6-6267</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mv8g-fhh6-6267",
            "title": "[django] Django user with hardcoded password created when running tests on Oracle",
            "date_modified": "2024-04-22T22:58:35.000Z",
            "date_published": "2022-05-17T00:27:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mv8g-fhh6-6267"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.8.x before 1.8.16, 1.9.x before 1.9.11, and 1.10.x before 1.10.3 use a hardcoded password for a temporary database user created when running tests with an Oracle database, which makes it easier for remote attackers to obtain access to the database server by leveraging failure to manually specify a password in the database settings TEST dictionary.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-9013\">https://nvd.nist.gov/vuln/detail/CVE-2016-9013</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/nov/01/security-releases/\">https://www.djangoproject.com/weblog/2016/nov/01/security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3835\">http://www.debian.org/security/2017/dsa-3835</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/94069\">http://www.securityfocus.com/bid/94069</a></li>\n<li><a href=\"http://www.securitytracker.com/id/1037159\">http://www.securitytracker.com/id/1037159</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-3115-1\">http://www.ubuntu.com/usn/USN-3115-1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mv8g-fhh6-6267\">https://github.com/advisories/GHSA-mv8g-fhh6-6267</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mv8g-fhh6-6267",
            "title": "[django] Django user with hardcoded password created when running tests on Oracle",
            "date_modified": "2024-04-22T22:58:35.000Z",
            "date_published": "2022-05-17T00:27:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mv8g-fhh6-6267"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.8.x before 1.8.16, 1.9.x before 1.9.11, and 1.10.x before 1.10.3 use a hardcoded password for a temporary database user created when running tests with an Oracle database, which makes it easier for remote attackers to obtain access to the database server by leveraging failure to manually specify a password in the database settings TEST dictionary.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-9013\">https://nvd.nist.gov/vuln/detail/CVE-2016-9013</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/nov/01/security-releases/\">https://www.djangoproject.com/weblog/2016/nov/01/security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3835\">http://www.debian.org/security/2017/dsa-3835</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/94069\">http://www.securityfocus.com/bid/94069</a></li>\n<li><a href=\"http://www.securitytracker.com/id/1037159\">http://www.securitytracker.com/id/1037159</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-3115-1\">http://www.ubuntu.com/usn/USN-3115-1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mv8g-fhh6-6267\">https://github.com/advisories/GHSA-mv8g-fhh6-6267</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mv8g-fhh6-6267",
            "title": "[django] Django user with hardcoded password created when running tests on Oracle",
            "date_modified": "2024-04-22T22:58:35.000Z",
            "date_published": "2022-05-17T00:27:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mv8g-fhh6-6267"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Vulnerability in the MySQL Connectors component of Oracle MySQL (subcomponent: Connector/Python). Supported versions that are affected are 2.1.5 and earlier. Easily \"exploitable\" vulnerability allows low privileged attacker with logon to the infrastructure where MySQL Connectors executes to compromise MySQL Connectors. Successful attacks of this vulnerability can result in unauthorized update, insert or delete access to some of MySQL Connectors accessible data. CVSS 3.0 Base Score 3.3 (Integrity impacts). CVSS Vector: (CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:L/A:N).</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-3590\">https://nvd.nist.gov/vuln/detail/CVE-2017-3590</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/security-advisory/cpuapr2017-3236618.html\">http://www.oracle.com/technetwork/security-advisory/cpuapr2017-3236618.html</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97840\">http://www.securityfocus.com/bid/97840</a></li>\n<li><a href=\"http://www.securitytracker.com/id/1038287\">http://www.securitytracker.com/id/1038287</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-2cf3-g243-hhfx\">https://github.com/advisories/GHSA-2cf3-g243-hhfx</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-2cf3-g243-hhfx",
            "title": "[mysql-connector-python] MySQL Connectors Privilege Escalation",
            "date_modified": "2024-04-22T22:58:16.000Z",
            "date_published": "2022-05-13T01:45:43.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-2cf3-g243-hhfx"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-0360\">https://nvd.nist.gov/vuln/detail/CVE-2017-0360</a></li>\n<li><a href=\"https://lists.debian.org/debian-security-announce/2017/msg00084.html\">https://lists.debian.org/debian-security-announce/2017/msg00084.html</a></li>\n<li><a href=\"http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8\">http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3826\">http://www.debian.org/security/2017/dsa-3826</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97489\">http://www.securityfocus.com/bid/97489</a></li>\n<li><a href=\"https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0\">https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7cwg-2575-3546\">https://github.com/advisories/GHSA-7cwg-2575-3546</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7cwg-2575-3546",
            "title": "[trytond] Tryton Information Disclosure Vulnerability",
            "date_modified": "2024-04-22T22:58:00.000Z",
            "date_published": "2022-05-13T01:39:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7cwg-2575-3546"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-0360\">https://nvd.nist.gov/vuln/detail/CVE-2017-0360</a></li>\n<li><a href=\"https://lists.debian.org/debian-security-announce/2017/msg00084.html\">https://lists.debian.org/debian-security-announce/2017/msg00084.html</a></li>\n<li><a href=\"http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8\">http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3826\">http://www.debian.org/security/2017/dsa-3826</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97489\">http://www.securityfocus.com/bid/97489</a></li>\n<li><a href=\"https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0\">https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7cwg-2575-3546\">https://github.com/advisories/GHSA-7cwg-2575-3546</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7cwg-2575-3546",
            "title": "[trytond] Tryton Information Disclosure Vulnerability",
            "date_modified": "2024-04-22T22:58:00.000Z",
            "date_published": "2022-05-13T01:39:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7cwg-2575-3546"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-0360\">https://nvd.nist.gov/vuln/detail/CVE-2017-0360</a></li>\n<li><a href=\"https://lists.debian.org/debian-security-announce/2017/msg00084.html\">https://lists.debian.org/debian-security-announce/2017/msg00084.html</a></li>\n<li><a href=\"http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8\">http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3826\">http://www.debian.org/security/2017/dsa-3826</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97489\">http://www.securityfocus.com/bid/97489</a></li>\n<li><a href=\"https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0\">https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7cwg-2575-3546\">https://github.com/advisories/GHSA-7cwg-2575-3546</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7cwg-2575-3546",
            "title": "[trytond] Tryton Information Disclosure Vulnerability",
            "date_modified": "2024-04-22T22:58:00.000Z",
            "date_published": "2022-05-13T01:39:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7cwg-2575-3546"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-0360\">https://nvd.nist.gov/vuln/detail/CVE-2017-0360</a></li>\n<li><a href=\"https://lists.debian.org/debian-security-announce/2017/msg00084.html\">https://lists.debian.org/debian-security-announce/2017/msg00084.html</a></li>\n<li><a href=\"http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8\">http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3826\">http://www.debian.org/security/2017/dsa-3826</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97489\">http://www.securityfocus.com/bid/97489</a></li>\n<li><a href=\"https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0\">https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7cwg-2575-3546\">https://github.com/advisories/GHSA-7cwg-2575-3546</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7cwg-2575-3546",
            "title": "[trytond] Tryton Information Disclosure Vulnerability",
            "date_modified": "2024-04-22T22:58:00.000Z",
            "date_published": "2022-05-13T01:39:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7cwg-2575-3546"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-0360\">https://nvd.nist.gov/vuln/detail/CVE-2017-0360</a></li>\n<li><a href=\"https://lists.debian.org/debian-security-announce/2017/msg00084.html\">https://lists.debian.org/debian-security-announce/2017/msg00084.html</a></li>\n<li><a href=\"http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8\">http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3826\">http://www.debian.org/security/2017/dsa-3826</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97489\">http://www.securityfocus.com/bid/97489</a></li>\n<li><a href=\"https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0\">https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7cwg-2575-3546\">https://github.com/advisories/GHSA-7cwg-2575-3546</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7cwg-2575-3546",
            "title": "[trytond] Tryton Information Disclosure Vulnerability",
            "date_modified": "2024-04-22T22:58:00.000Z",
            "date_published": "2022-05-13T01:39:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7cwg-2575-3546"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-0360\">https://nvd.nist.gov/vuln/detail/CVE-2017-0360</a></li>\n<li><a href=\"https://lists.debian.org/debian-security-announce/2017/msg00084.html\">https://lists.debian.org/debian-security-announce/2017/msg00084.html</a></li>\n<li><a href=\"http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8\">http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3826\">http://www.debian.org/security/2017/dsa-3826</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97489\">http://www.securityfocus.com/bid/97489</a></li>\n<li><a href=\"https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0\">https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7cwg-2575-3546\">https://github.com/advisories/GHSA-7cwg-2575-3546</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7cwg-2575-3546",
            "title": "[trytond] Tryton Information Disclosure Vulnerability",
            "date_modified": "2024-04-22T22:58:00.000Z",
            "date_published": "2022-05-13T01:39:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7cwg-2575-3546"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>file_open in Tryton 3.x and 4.x through 4.2.2 allows remote authenticated users with certain permissions to read arbitrary files via a \"same root name but with a suffix\" attack. NOTE: This vulnerability exists because of an incomplete fix for CVE-2016-1242.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-0360\">https://nvd.nist.gov/vuln/detail/CVE-2017-0360</a></li>\n<li><a href=\"https://lists.debian.org/debian-security-announce/2017/msg00084.html\">https://lists.debian.org/debian-security-announce/2017/msg00084.html</a></li>\n<li><a href=\"http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8\">http://hg.tryton.org/trytond?cmd=changeset;node=472510fdc6f8</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3826\">http://www.debian.org/security/2017/dsa-3826</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/97489\">http://www.securityfocus.com/bid/97489</a></li>\n<li><a href=\"https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0\">https://github.com/tryton/trytond/commit/30e978593733385db3144f8c583eeb4679575cf0</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7cwg-2575-3546\">https://github.com/advisories/GHSA-7cwg-2575-3546</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7cwg-2575-3546",
            "title": "[trytond] Tryton Information Disclosure Vulnerability",
            "date_modified": "2024-04-22T22:58:00.000Z",
            "date_published": "2022-05-13T01:39:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7cwg-2575-3546"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in NFStream 5.2.0. Because some allocated modules are not correctly freed, if the nfstream object is directly destroyed without being used after it is created, it will cause a memory leak that may result in a local denial of service (DoS).</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-25340\">https://nvd.nist.gov/vuln/detail/CVE-2020-25340</a></li>\n<li><a href=\"https://github.com/ntop/nDPI/issues/994\">https://github.com/ntop/nDPI/issues/994</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-whmq-cfm5-j8mj\">https://github.com/advisories/GHSA-whmq-cfm5-j8mj</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-whmq-cfm5-j8mj",
            "title": "[nfstream] NFStream Local Denial of Service (DoS)",
            "date_modified": "2024-04-22T22:57:25.000Z",
            "date_published": "2022-05-24T17:42:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-whmq-cfm5-j8mj"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In some conditions, a snap package built by snapcraft includes the current directory in <code>LD_LIBRARY_PATH</code>, allowing a malicious snap to gain code execution within the context of another snap if both plug the home interface or similar. This issue affects snapcraft versions prior to 4.4.4, prior to 2.43.1+16.04.1, and prior to 2.43.1+18.04.1.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-27348\">https://nvd.nist.gov/vuln/detail/CVE-2020-27348</a></li>\n<li><a href=\"https://github.com/snapcore/snapcraft/pull/3345\">https://github.com/snapcore/snapcraft/pull/3345</a></li>\n<li><a href=\"https://bugs.launchpad.net/bugs/1901572\">https://bugs.launchpad.net/bugs/1901572</a></li>\n<li><a href=\"https://usn.ubuntu.com/usn/usn-4661-1\">https://usn.ubuntu.com/usn/usn-4661-1</a></li>\n<li><a href=\"https://github.com/snapcore/snapcraft/commit/a0ceca9d531a34c979251030ed67b5fa2abfdd9a\">https://github.com/snapcore/snapcraft/commit/a0ceca9d531a34c979251030ed67b5fa2abfdd9a</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-qxm5-vx5j-pp6w\">https://github.com/advisories/GHSA-qxm5-vx5j-pp6w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-qxm5-vx5j-pp6w",
            "title": "[snapcraft] snapcraft Access Restriction Bypass",
            "date_modified": "2024-04-22T22:57:15.000Z",
            "date_published": "2022-05-24T17:35:23.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-qxm5-vx5j-pp6w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>scikit-learn (aka sklearn) through 0.23.0 can unserialize and execute commands from an untrusted file that is passed to the <code>joblib.load()</code> function, if <code>__reduce__</code> makes an <code>os.system call</code>.\nNOTE: third parties dispute this issue because the joblib.load() function is documented as unsafe and it is the user's responsibility to use the function in a secure manner.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-13092\">https://nvd.nist.gov/vuln/detail/CVE-2020-13092</a></li>\n<li><a href=\"https://github.com/0FuzzingQ/vuln/blob/master/sklearn%20unserialize.md\">https://github.com/0FuzzingQ/vuln/blob/master/sklearn%20unserialize.md</a></li>\n<li><a href=\"https://scikit-learn.org/stable/modules/model_persistence.html#security-maintainability-limitations\">https://scikit-learn.org/stable/modules/model_persistence.html#security-maintainability-limitations</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-jjw5-xxj6-pcv5\">https://github.com/advisories/GHSA-jjw5-xxj6-pcv5</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-jjw5-xxj6-pcv5",
            "title": "[scikit-learn] scikit-learn Deserialization of Untrusted Data",
            "date_modified": "2024-04-22T22:55:45.000Z",
            "date_published": "2022-05-24T17:17:59.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-jjw5-xxj6-pcv5"
            },
            "tags": [
                "severity"
            ]
        }
    ]
}